src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -166,7 +168,7 @@ CHANGELOG
    The default value will be `state_machine` in Symfony 4.0.
  * Deprecated the `CompilerDebugDumpPass` class
  * Deprecated the "framework.trusted_proxies" configuration option and the corresponding "kernel.trusted_proxies" parameter
- * Added a new new version strategy option called json_manifest_path
+ * Added a new version strategy option called json_manifest_path
```

Comment:
this should be fixed in 3.4 instead

Reply:
But if you keep it here and target 4.2 as a bugfix, please target 4.2 brach instead of master branch 👌🏻

Reply:
But if you keep it here and target 4.2 as a bugfix, please target 4.2 brach instead of master branch 👌🏻

Reply:
If you expand the code you'll see that this line belong to 3.3.0 section, hence this change should be reverted and fixed in another PR targering 3.4 branch.

Reply:
done see https://github.com/symfony/symfony/pull/31137/files